uiowa/uiowa
# Iisc deliverables

Resolves #5406 

This does also enable the 'description' field for File type media entities on the platform, allowing the description to be used in places where a media field is being displayed. If there is no description (default for any pre-existing media entities with this update), the display will continue to use the filename when displaying the rendered entity.





# How to test


`git checkout iisc-deliverables && git pull`
`ddev blt ds --site=iisc.uiowa.edu`
Create a project with a deliverable which uses a description in the media entity
Check the project display, and see that it displays the description, not the filename

Run the migrations up to at least a few projects
Check a migrated project's display, and see that it displays the human readable name, not the filename

Try syncing some custom split sites like admissions, and check that it completes without config errors
Try opening and editing an existing media entity, and adding new media entity, and check that they save properly with or without a description
